app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -0,0 +1,32 @@
+<div id="account-recovery-key">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery key{{/t}}</h2>
+
+        <p class="description">
+            {{#unsafeTranslate}}<span class="note">NOTE: </span>Keep your recovery key in a safe place. You will not be able to access this page again. You can download or print your key or email it to someone you trust{{/unsafeTranslate}}
+        </p>
+
+        <div class="recovery-key-container">
+            <div class="recovery-key">{{recoveryKey}}</div>
```

Comment:
The recovery key could be split into sections of 4 like a credit card to make it easier to write:

<img width="339" alt="screen shot 2018-07-31 at 16 38 08" src="https://user-images.githubusercontent.com/848085/43470286-1dde0d88-94e0-11e8-8895-6b8580ac05e9.png">

vs

<img width="362" alt="screen shot 2018-07-31 at 16 39 07" src="https://user-images.githubusercontent.com/848085/43470351-44e21168-94e0-11e8-9ea1-7929ea9163be.png">

Reply:
This change looks great, thanks!